tests/conftest.py

```diff
@@ -248,33 +248,35 @@ def patched_get_exit_signal(pair, timeframe, dataframe, is_short):
     freqtrade.exchange.refresh_latest_ohlcv = lambda p: None
 
 
-def create_mock_trades(fee, is_short: bool = False, use_db: bool = True):
+def create_mock_trades(fee, is_short: Optional[bool] = False, use_db: bool = True):
     """
     Create some fake trades ...
+    :param is_short: Optional bool, None creates a mix of long and short trades.
     """
     def add_trade(trade):
         if use_db:
             Trade.query.session.add(trade)
         else:
             LocalTrade.add_bt_trade(trade)
-
+    is_short1 = is_short if is_short is not None else True
+    is_short2 = is_short if is_short is not None else False
```

Comment:
Why split them up? My intention was to test a bunch of long trades, or else test a bunch of short trades

Reply:
@xmatthias and I felt that we should also test mixed trades (longs and shorts)
I can create another test case for this if we want to test (bunch longs bunch shorts) then mixed trades.

Reply:
For RPC methods, testing only long or short is a strange way to test things, as it will never test a mix of trades, where some trades are longs, and some are shorts (which is way more realistic).

So to actually check if the calculations are correct, we'll need a "Middle ground" - where we have both long and short tests created in an easy way.
While we could create another method for that - i don't really see a point in it, as python allows a boolean value to be True or False or None.